src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,20 @@
+@import "./variables";
+
+@mixin element-size ($height , $width) {
+  height: $height;
+  width: $width;
+}
+
+@mixin position ($left , $top) {
+  left: $left;
+  top: $top;
+}
+@mixin default-set {
```

Comment:
Add **box-sizing: border-box**; to reset.css and specify elements that need to have this property. 
Then you could add **position: absolute**; directly to elements